### PR TITLE
Feature/dcwc1426 apply for cards

### PR DESCRIPTION
**Pull request by amirpourpanahi**
### Description

List of proposed changes:

- Developing "Apply for" cards for multiple CPP benefits
- Adding SubBenefitCode file
- Unit test is updated for sub-benefits

### CODE REVIEW

File: components/molecules/BenefitApplicationCard.js [hunk 1]
```diff
@@ -37,7 +45,11 @@ const BenefitApplicationCard = (props) => {
               href={props.benefitApplication.learnMoreLink}
               className="text-xl underline hover:text-bright-blue-solid"
             >
-              {props.benefitApplication.benefitName}
+              {typeof props.benefitApplication.benefitSubType === 'undefined'
+                ? t.applyFor + ' ' + props.benefitApplication.benefitName
```
Comment (nabil386) [inline components/molecules/BenefitApplicationCard.js:49]: maybe a function that takes 3 params and returns the string already formatted would be more efficient

**Review by nabil386 — CHANGES_REQUESTED** [review]

File: components/molecules/BenefitApplicationCard.js (hunk 1 above)
Comment (amirpourpanahi) [reply components/molecules/BenefitApplicationCard.js]: I added function to return string but I didn't add 3 parameters to use it for both "Apply" button and "Learn More" link.

**Review by nabil386 — APPROVED** [review]

File: components/molecules/BenefitApplicationCard.js [hunk 2]
```diff
@@ -19,12 +20,19 @@ const BenefitApplicationCard = (props) => {
     <div className="benefit-card px-4 md:px-6 py-7">
       <div className="mx-auto sm:grid sm:grid-cols-4 sm:divide-x-2">
         <div
-          id={`${props.benefitApplication.benefitType}-app-card`}
+          id={`${
+            props.benefitApplication.benefitType +
+            '-' +
```
[the remark this answers is not in the document]
Comment (amirpourpanahi) [reply components/molecules/BenefitApplicationCard.js:33]: Done

File: components/molecules/BenefitApplicationCard.js [hunk 3]
```diff
@@ -19,12 +20,19 @@ const BenefitApplicationCard = (props) => {
...
+            props.benefitApplication.benefitSubType
+          }-app-card`}
           className="col-span-1 py-4 md:px-0 lg:px-3"
         >
           <div className="font-bold font-display text-4xl sm:text-2xl lg:text-4xl mb-2 w-44 sm:w-32 lg:w-44">
             {props.benefitApplication.benefitName}
           </div>
+          <div className="font-bold font-display text-md sm:text-md lg:text-md mb-2 w-44 sm:w-32 lg:w-44">
+            {t[props.benefitApplication.benefitSubType]}
```
[the remark this answers is not in the document]
Comment (amirpourpanahi) [reply components/molecules/BenefitApplicationCard.js:42]: Done

**Review by habibaz — APPROVED** [review]

**Pull request merged**